### ISSUE TITLE
File with info about contributors and maintainers

### ISSUE THREAD

**Issue by bebatut** [galaxyproject/training-material#490]
Currently in the metadata, the info about the contributors (full name, GitHub username, email, ...) are added everytime. Then the email is often empty. 

One solution for that would be to have at the root (or even better in an `_data` folder) a YAML file with the list of every contributors and maintainers with the useful info and with the GitHub username as key, like:

```
bebatut:
        name: "Bérénice Batut"
        email: berenice.batut@gmail.com
shiltemann:
        name: "Saskia Hiltemann"
        email: 
```
and in the metadata files just having a list with the GitHub usernames of the contributors/maintainers: 

```
contributors:
- bebatut
- shiltemann
```

We will need to update the layouts ([`_layouts/topic.html`](https://github.com/galaxyproject/training-material/blob/master/_layouts/topic.html), [`_layouts/base_slides.html`](https://github.com/galaxyproject/training-material/blob/master/_layouts/base_slides.html) and [`_layouts/tutorial_hands_on.html`](https://github.com/galaxyproject/training-material/blob/master/_layouts/tutorial_hands_on.html)) to parse the list and the correct information when needed.

But it will be easier to maintain and we can also add more information

What do you think?

**Comment by shiltemann** [comment]
yes, definitely, this was on my wishlist as well, makes it lot easier for contributers to update their contact information when it changes etc :+1:

**Comment by bebatut** [comment]
We can do that once we have merged #484

**Comment by willdurand** [comment]
fixed.
